.test/library/ubuntu

```diff
@@ -0,0 +1,15 @@
+# https://github.com/docker-library/official-images/blob/fe9c059402181390eac083cbdd7229b5d123236e/library/ubuntu but intentionally slimmed down (just "latest" on one architecture, no email addresses)
+
+Maintainers: Tomáš Virtus (@woky), Cristóvão Cordeiro (@cjdcordeiro)
+GitRepo: https://git.launchpad.net/cloud-images/+oci/ubuntu-base
+GitCommit: fa42be9027eccb928a1f0f43d95ffd9a45d36737
+Builder: oci-import
+File: index.json
```

Comment:
The `oci-import` builder is a bashbrew construction that uses OCI layouts, but does not mandate them explicitly -- see https://github.com/docker-library/bashbrew/blob/5152c0df682515cbe7ac62b68bcea4278856429f/cmd/bashbrew/oci-builder.go#L117-L134

If `File:` is not set to the literal string `index.json`, then it is assumed to point to a JSON file that contains a single OCI content descriptor for the image manifest (which will then be looked up under `blobs/sha256/xxxx`).

See also https://github.com/docker-library/meta-scripts/pull/20/files#diff-a64244565ecee6714e591bafe1b0771f08c5217287fb88a6ff0e9f1457eb8ac3R296-R299 for where in this PR those get "upgraded" into a correct `index.json` so that we don't have to handle both cases exceptionally here.

(_Technically_, the code I've written there to "upgrade" that case to `index.json` would break if the object under `blobs/sha256/xxx` is a symlink to `index.json` at the root of the OCI layout, but given we've established that the format is "OCI layout", it's fair for us to assume that `index.json` at the root has special meaning and whatever is happening in that theoretical case is wrong.)